service/awsconfig/v10/adapter/instance.go

```diff
@@ -1,55 +0,0 @@
-package adapter
```

Comment:
I renamed the adapter to the plural form and changed a lot which is why the diff is weird.

Reply:
The change from `instance` to `instances` I find confusing. As we only create a single master per guest cluster.

This will need to change later for HA masters but that work is not scheduled soon and we'll probably need to use an autoscaling group.

Reply:
Changed it to singular form again.